src/main/kotlin/com/sourcegraph/cody/error/CodyErrorSubmitter.kt

```diff
@@ -67,7 +66,7 @@ class CodyErrorSubmitter : ErrorReportSubmitter() {
 
   private fun getTitle(throwableText: String): String {
     val title = trimPostfix(throwableText.lines().first(), 128)
```

Comment:
Don't use two lines where one will do.

```suggestion
    return trimPostfix(throwableText.lines().first(), 128)
```